.eslintrc.js

```diff
@@ -0,0 +1,36 @@
+module.exports = {
+  root: true,
+
+  env: {
+    node: true,
+  },
+
+  extends: [
+    'plugin:vue/recommended',
+    '@vue/airbnb',
+    '@vue/typescript',
+    'plugin:vue/recommended',
+    '@vue/typescript/recommended',
+  ],
+
+  plugins: [
+    'vuetify',
+  ],
+
+  rules: {
+    'no-console': process.env.NODE_ENV === 'production' ? 'warn' : 'off',
+    'no-debugger': process.env.NODE_ENV === 'production' ? 'warn' : 'off',
+    'max-len': ['off'],
+    'import/prefer-default-export': ['off'],
+    'no-underscore-dangle': ['off'],
```

Comment:
Personally, I think this is a good rule to leave as a default. Underscores just aren't needed to indicate privateness, etc., since JavaScript/TypeScript have good export mechanisms for "publishing" functions from a module.

Reply:
I'm mostly using it for unused params in arrow functions, not just privateness. You still think that we should remove it?

Reply:
If you don't use underscore for unused parameters in an arrow function, does some other rule get triggered? I don't think you need the underscore style for that purpose; if an arrow function is using the second argument but not the first, I think you should just name the first argument normally and then not use it (my gut tells me eslint should allow that, since it's a common situation when passing arrow functions to a framework).

Reply:
Looks like my memory was wrong, it was for handling multinet columns from Arango, such as _key, _from, _to, etc. Here's an example of the warning that is thrown without disabling this rule:

```
error: Unexpected dangling '_' in '_from' (no-underscore-dangle) at src/lib/multinet.ts:77:69:
  75 |   links.forEach((link) => {
  76 |     nodes.filter((d: Node) => d._id === link._from)[0].neighbors.push(link._to);
> 77 |     nodes.filter((d: Node) => d._id === link._to)[0].neighbors.push(link._from);
     |                                                                     ^
  78 |   });
  79 |   return nodes;
  80 | }
```

Reply:
Ah yeah, this is annoying. I think your rule is fine, though I guess I'd toy with using the subscripting form for these uses (e.g., `link['_to']'`etc. If you keep your eslint rule, I'd just request that you leave a comment explaining why it's there.

Reply:
Fixed in 831e857